pgjdbc/src/main/java/org/postgresql/core/NativeQuery.java

```diff
@@ -80,13 +94,8 @@ public static String bindName(int index) {
     return index < BIND_NAMES.length ? BIND_NAMES[index] : "$" + index;
   }
 
-  public static StringBuilder appendBindName(StringBuilder sb, int index) {
-    if (index < BIND_NAMES.length) {
-      return sb.append(bindName(index));
-    }
-    sb.append('$');
-    sb.append(index);
-    return sb;
+  public static void appendBindName(StringBuilder sb, int index) {
+    sb.append(bindName(index));
```

Comment:
why is this change?

Reply:
Techically speaking, the previous call was appending `$` and the integer with two separate `StringBuidler#append` calls and moving to `bindName`would force string concatenation within `bindName`.
Of course, it is not super-critical for performance, however, at the same time, I do not see why the old code was bothering you, so why change something that was there?